src/scenes/MemberListPage/MemberListTier.tsx

```diff
@@ -15,21 +18,35 @@ function getPreviewDirection(index: number) {
 }
 
 type Props = {
-  tier: MembershipTier;
+  tierRef: MemberListTierFragment$key;
 };
 
-function MemberListTier({ tier }: Props) {
-  const { fadeUsernames } = useMemberListPageState();
+function MemberListTier({ tierRef }: Props) {
+  const tier = useFragment(
+    graphql`
+      fragment MemberListTierFragment on MembershipTier {
+        name
+        owners {
+          id
+          user @required(action: NONE) {
+            username @required(action: NONE)
+          }
+
+          ...MemberListOwnerFragment
+        }
+      }
+    `,
+    tierRef
+  );
+
+  const { fadeUsernames, searchQuery } = useMemberListPageState();
 
-  const { searchQuery } = useMemberListPageState();
   const sortedOwners = useMemo(() => {
-    if (!tier.owners) {
-      return [];
-    }
+    const nonNullOwners = removeNullValues(tier.owners ?? []);
+
+    nonNullOwners.sort((a, b) => a.user.username.localeCompare(b.user.username));
```

Comment:
it looks like we lost the username.toLowerCase() here. some usernames are capitalized and we want to ignore case when sorting the list alphabetically